extensions/podman/package.json

```diff
@@ -379,6 +379,7 @@
   "dependencies": {
     "@ltd/j-toml": "^1.38.0",
     "@podman-desktop/api": "^0.0.1",
+    "@podman-desktop/podman-extension-api": "^0.0.1",
```

Comment:
I am wondering if we could improve the name by adding a `/`

Here is my idea

`@podman-desktop/extension-api/podman`

what do you think @benoitf ?

Reply:
AFAIK npmjs package are only `@org-name/package-name`

Reply:
https://docs.npmjs.com/cli/v10/using-npm/scope#publishing-scoped-packages

Reply:
Oh that's so said